presqt/api_v1/views/status/status.py

```diff
@@ -0,0 +1,80 @@
+import requests
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+class StatusCollection(APIView):
+    """
+    **Supported HTTP Methods**
+
+    * Get: Retrieve summary representations of all Targets's status.
+    """
+
+    required_scopes = ["read"]
+
+    def get(self, request):
+        """
+        Retrieve all Targets.
+
+        Returns
+        -------
+        200 : OK
+        A list-like JSON representation of each Target's status.
+        [
+            {
+                "service": "osf",
+                "status": "200"
+            },
+            {
+                "service": "osf",
+                "status": "offline"
+                "detail": "
+            },
+        ]
+        """
+
+        # TODO: where should this go?
+        config = {
+            "osf": "https://api.osf.io/v2/nodes/",
+            "curate_nd": "https://curate.nd.edu/api/items",
+            "github": "https://api.github.com/repositories",
+            "zenodo": "https://zenodo.org/api/records/",
+            "gitlab": "https://gitlab.com/api/v4/projects",
+        }
```

Comment:
add status_endpoint to targets.json
also update:
* targets.json schema,
* serializer --? don't display
* targets keys documentation
* tests, target tests

Reply:
also:
- [ ] add_new_target.py